### DISCUSSION TITLE
Cypress - Click on an element only if it is clickable

### DISCUSSION THREAD

**Discussion by Sibasish9088** [cypress-io/cypress#17504] [Questions and Help]
Hi,

Is there a method in Cypress to check if an element is clickable and then click on it?

An example of what I am requesting information in would be:

`while (cy.get(<locator string>).isClickable()) {
cy.get(<locator string>).click();
//and some other instructions to follow;
}`

An example scenario for this would be the paginator functionality of a webpage.
So this function or method would enable me to click on the 'next page' or 'previous page' icons until I can go no further.

Hope I was able to portray it correctly! ;)

**Comment by bahmutov** [comment]
See https://github.com/bahmutov/cypress-recurse
